docs/lang/articles/debug/debugging.md

```diff
@@ -173,7 +179,7 @@ def test() -> ti.i32:
 print(test())
 ```
 
-The code snippet above raises a `TaichiAssertionError`, indicating that you are trying to access a field with improper indices.
+The code snippet above raises a `TaichiAssertionError`, because we are trying to access element of indices `[0, 73]` in the field `f`, while the second dimension of `f` is only 32.
```

Comment:
```suggestion
The code snippet above raises a `TaichiAssertionError` because you are trying to access elements from a field of shape (32, 32) with indices in the range of `[0, 73]`.
```

Reply:
with indices `[0, 73]` is OK, it's not in the range of `[0, 73]`.

Reply:
Updated :)